sdks/java/maven-archetypes/examples/src/main/resources/archetype-resources/src/main/java/WindowedWordCount.java

```diff
@@ -17,16 +17,16 @@
  */
 package ${package};
 
-import ${package}.common.ExampleBigQueryTableOptions;
-import ${package}.common.ExampleOptions;
-import ${package}.common.ExampleUtils;
 import com.google.api.services.bigquery.model.TableFieldSchema;
 import com.google.api.services.bigquery.model.TableReference;
 import com.google.api.services.bigquery.model.TableRow;
 import com.google.api.services.bigquery.model.TableSchema;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import ${package}.common.ExampleBigQueryTableOptions;
```

Comment:
You mean the reordering? Previously, we had manually reordered it to be first. For a user who just generated the archetype, that is kind of friendly, but probably irrelevant as they won't look at imports and/or their IDE will reorder it. So I didn't include that complexity in the script.